src/data/featured.json

```diff
@@ -1,8 +1,8 @@
 {
-	"title": "ElectronJS with Raspberry Pi Made Easy As Pie",
-	"tags": ["tutorial"],
-	"text": "Showcasing the opportunities of ElectronJS with a Raspberry Pi to create digital signage and interactive displays, including example projects!",
-	"link": "https://resin.io/blog/electronjs-the-ultimate-guide/",
-	"image": "http://i.imgur.com/8GWASHi.jpg",
-	"button": "Deploy it"
+	"title": "ResinOS 2.0 Released in Production",
```

Comment:
Let's edit all the titles to start with a verb that applies to the audience - e.g. "Deploy resinOS 2.0 to your fleet"